tests/test_her.py

```diff
@@ -73,11 +80,16 @@ def test_her(model_class, online_sampling, image_obs_space):
     ],
 )
 @pytest.mark.parametrize("online_sampling", [True, False])
-def test_goal_selection_strategy(goal_selection_strategy, online_sampling):
+def test_goal_selection_strategy(n_envs, goal_selection_strategy, online_sampling):
     """
     Test different goal strategies.
     """
-    env = BitFlippingEnv(continuous=True)
+    if n_envs > 1 and not online_sampling:
+        pytest.skip("Offline sampling is not compatible with multiprocessing")
+
+    env_fn = lambda: BitFlippingEnv(continuous=True)
```

Comment:
is there a reason you reverted to lamndas? (see linter warnnigs)

Reply:
None, I did not see your commit. I am reverting this change.